ion_hash_tests.ion

```diff
@@ -922,56 +908,6 @@ sexpCount128::{
-    int:5,
-    neg_int:-6,
-    float:4.9e-324,
-    decimal:123.45,
-    timestamp:2017-01-01T00:00:00-00:00,
-    symbol:hello,
-    string:"hello",
-    clob:{{"hello"}},
-    blob:{{aGVsbG8=}},
-    list:[1,2,3],
-    sexp:(1 2 3),
-    struct:{},
-    annotated_value:hello::{},
-  },
-  expect:{ identity:(
-    final_digest::(
-      0x0b 0xd0
-        0x0c 0x0b 0x70 0x61 0x6e 0x6e 0x6f 0x74 0x61 0x74 0x65 0x64 0x5f 0x76 0x61 0x6c 0x75 0x65 0x0c 0x0e
-             0x0c 0x0b 0xe0 0x0c 0x0b 0x70 0x68 0x65 0x6c 0x6c 0x6f 0x0c 0x0e 0x0c 0x0b 0xd0 0x0c 0x0e 0x0c 0x0e                 // annotated_value:hello::{}
-        0x0c 0x0b 0x70 0x62 0x6c 0x6f 0x62 0x0c 0x0e 0x0c 0x0b 0xa0 0x68 0x65 0x6c 0x6c 0x6f 0x0c 0x0e                           // blob:{{aGVsbG8=}}
-        0x0c 0x0b 0x70 0x62 0x6f 0x6f 0x6c 0x0c 0x0e 0x0c 0x0b 0x10 0x0c 0x0e                                                    // bool:false
-        0x0c 0x0b 0x70 0x63 0x6c 0x6f 0x62 0x0c 0x0e 0x0c 0x0b 0x90 0x68 0x65 0x6c 0x6c 0x6f 0x0c 0x0e                           // clob:{{"hello"}}
-        0x0c 0x0b 0x70 0x64 0x65 0x63 0x69 0x6d 0x61 0x6c 0x0c 0x0e 0x0c 0x0b 0x50 0xc2 0x30 0x39 0x0c 0x0e                      // decimal:123.45
-        0x0c 0x0b 0x70 0x66 0x6c 0x6f 0x61 0x74 0x0c 0x0e 0x0c 0x0b 0x40 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x01 0x0c 0x0e       // float:4-9e-324
-        0x0c 0x0b 0x70 0x69 0x6e 0x74 0x0c 0x0e 0x0c 0x0b 0x20 0x05 0x0c 0x0e                                                    // int:5
-        0x0c 0x0b 0x70 0x6c 0x69 0x73 0x74 0x0c 0x0e
-             0x0c 0x0b 0xb0 0x0c 0x0b 0x20 0x01 0x0c 0x0e 0x0c 0x0b 0x20 0x02 0x0c 0x0e 0x0c 0x0b 0x20 0x03 0x0c 0x0e 0x0c 0x0e  // list:[1,2,3]
-        0x0c 0x0b 0x70 0x6e 0x65 0x67 0x5f 0x69 0x6e 0x74 0x0c 0x0e 0x0c 0x0b 0x30 0x06 0x0c 0x0e                                // neg_int:-6
-        0x0c 0x0b 0x70 0x6e 0x75 0x6c 0x6c 0x0c 0x0e 0x0c 0x0b 0x0f 0x0c 0x0e                                                    // null:null
-        0x0c 0x0b 0x70 0x73 0x65 0x78 0x70 0x0c 0x0e
-             0x0c 0x0b 0xc0 0x0c 0x0b 0x20 0x01 0x0c 0x0e 0x0c 0x0b 0x20 0x02 0x0c 0x0e 0x0c 0x0b 0x20 0x03 0x0c 0x0e 0x0c 0x0e  // sexp:[1,2,3]
-        0x0c 0x0b 0x70 0x73 0x74 0x72 0x69 0x6e 0x67 0x0c 0x0e 0x0c 0x0b 0x80 0x68 0x65 0x6c 0x6c 0x6f 0x0c 0x0e                 // string:"hello"
-        0x0c 0x0b 0x70 0x73 0x74 0x72 0x75 0x63 0x74 0x0c 0x0e 0x0c 0x0b 0xd0 0x0c 0x0e                                          // struct:{}
-        0x0c 0x0b 0x70 0x73 0x79 0x6d 0x62 0x6f 0x6c 0x0c 0x0e 0x0c 0x0b 0x70 0x68 0x65 0x6c 0x6c 0x6f 0x0c 0x0e                 // symbol:hello
-        0x0c 0x0b 0x70 0x74 0x69 0x6d 0x65 0x73 0x74 0x61 0x6d 0x70 0x0c 0x0e
-             0x0c 0x0b 0x60 0xc0 0x0f 0xe1 0x81 0x81 0x80 0x80 0x80 0x0c 0x0e                                                    // timestamp:2017-01-01T00:00:00-00:00
-      0x0e
-    )
-  ) }
-}
-{
-  ion:{float_zero:0e0},    // verify this special value gets added to a struct
```

Comment:
I thought we did not cover negative 0 but this is float 0 is this not supported?

Reply:
My bad, I removed it out of the commented out section